modules/instrument_manager/help/instrument_manager.md

```diff
@@ -1,54 +1,15 @@
 # Instrument manager
 
-## Uploading Instruments
+This module allows you to view the status of instruments that are installed in LORIS. If configured, the module will contain an **Upload** tab, where you can install new instruments created with the Instrument Builder. 
 
-The instrument manager provides the ability to view the status of
-instruments that are installed in LORIS and, if configured, the
-ability to install new instruments created with the instrument
-builder.
+## View Installed Instruments
 
-If configured, the top of the page displays an "Upload Instrument"
-section where a user can click "Browse" to select a LINST file from
-their computer.  After selecting a valid LINST file, clicking
-"Install Instrument" will install the instrument into LORIS (but
-it must still be manually inserted into a study specific test battery.)
+In the **Browse** tab, use the *Selection Filter* section to search for a specific instrument or instruments. The resulting table will display details on the instrument, including whether or not the instrument SQL table exists and is valid. 
 
-## Viewing Installed Instruments
+**Note**: the *Table Valid* and *Pages Valid* columns can only be determined for LINST instruments (those created in the instrument builder), as there is no way to extract the “proper” values from instruments coded in PHP. The *Instrument Type* column displays whether the instrument was created in the instrument builder or coded in PHP. 
 
-Below the "Upload Instrument" section is a table displaying all
-instruments which are currently installed in LORIS. For each
-instrument it displays the columns:
+## Upload New Instrument
 
-*Instrument*: This provides the name of the instrument (as given
-in the `test_names` table).
```

Comment:
pls add back : ... as in the test names table

Reply:
done